### ISSUE TITLE
Add a way to share mods/ressources pack/shader list from a configuration

### ISSUE THREAD

**Issue by Dysta** [PolyMC/PolyMC#1640]
### Role

I play modded Minecraft with friends

### Suggestion

I want to easily share a mods list with them

### Benefit

No version mismatch, fast way to share a list of mods, more custom modpack

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

This idea is from the Thunderstore Manager. When you play a game, you can share a modlist from a code or a file then the loader download them for you. For your friends, its easy to import your modlist without the pain to create a modpack and share it. Also, it avoid sharing a full folder that can be upto +500Mo.

In the Thunderstore Manager its like this:

You can import a new profile/update a current profile from a code or a file like this
![image](https://github.com/PolyMC/PolyMC/assets/25008828/cb7066b0-b157-430e-8f10-bc865b1385ac)

Same for sharing your profile configuration, just click the share button then it gives you a code to share with your friends
![image](https://github.com/PolyMC/PolyMC/assets/25008828/633e3268-e37a-4ce1-9b7e-b7564fd4446d)

or go to settings to have a way to export as a file
![image](https://github.com/PolyMC/PolyMC/assets/25008828/716f2c77-c28e-42f3-99cd-35104932b130)

The file is in a custom format but I imagine that it contain the mod list and all others related info.

**Comment by Nik-mmzd** [comment]
+1
I think it can be resolved by adding "export to mrpack" (as most appropriate format) feature

**Comment by binex-dsk** [comment]
We can just as easily implement a global folder within the PolyMC config directory that carries all mods, shaders, etc. used and allow users to select them from instance pages.

**Comment by ghost** [comment]
We can just as easily implement a global folder within the PolyMC config directory that carries all mods, shaders, etc. used and allow users to select them from instance pages.

**Comment by Sneedplex** [comment]
Impressive, very sneed.

**Comment by Nik-mmzd** [comment]
@binex-dsk I don't think so: mods, resourcepacks and shaderpacks tend to be updated, and this folder would be a waste of disk space, as it would mostly contain out-of-date data.

**Comment by Dysta** [comment]
Hello, any update ?

**Comment by ghost** [comment]
> Hello, any update ? 

Most of us don't have oodles of time to do this stuff, as we have education, jobs, graduation, holidays, etc. (unlike the Prism developers evidently).

I may stab at this soon.

**Comment by EchoEllet** [comment]
Use any script that syncs the minecraft instance resources like the mods, etc..
Most launchers already have pre launch commands to run the script

**Comment by dxcvvxd** [comment]
> I may stab at this soon.

any updates?
